README.md

```diff
@@ -1,140 +1,158 @@
 
-**Q: Why is the identifier of a PAC-ID located in the query string?**
+- Add `id segment`s that partition by resource type or domain. (e.g. a CDS can only provide more information about Chromatography runs, but not MS runs)
 
-**A:** Query strings are easily accessible on the server side. In the textual representation, the query string is clearly separated from the rest, left of the question mark ("?").
+- Using the key/value syntax (`id segment key` `:` `id segment value`) with [well-known `id segment keys`](well-known-id-segment-keys.md) enables `PAC-ID` routing in generic contexts.
 
-**Q: Why is the subdomain "pac." used?**
+## PAC-ID Extension
 
-**A:** First of all, it's easy recognizable as a PAC-ID with this prefix/subdomain. Also, the `issuer` of a PAC-ID usually corresponds to the same domain name as the product manufacturer who distributed a PAC-ID. As most manufacturers are running a "www" web site on their main domain, usually under control of the marketing department, it is much easier to release a PAC-ID resolver on a subdomain than on a specific path like "/pac-id". See also question above on "Why is a PAC-ID represented as an https URL?".
+`PAC-ID`s can be extended with custom information after a `*` character. As stated above, the `*` character and subsequent characters are not part of the `PAC-ID` and can therefore be used for custom extensions. 
 
-**Q: Why not using the ["tag:" scheme](https://en.wikipedia.org/wiki/Tag_URI_scheme)?**
+A well-known extension is the [T-REX](https://github.com/ApiniLabs/T-REX).
 
-**A:** The `tag:` URI scheme would not allow to form valid URLs. This was considered a major disadvantage, see also question above on "Why is a PAC-ID represented as an https URL?".
+## Terminology Used
 
-**Q: Is the usage of PAC-IDs limited to life science / laboratories?**
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt) "Key words for use in RFCs to Indicate Requirement Levels".
 
-**A:** Not by design. In principle PAC-IDs could be used everywhere where real world objects need to be unambiguously identified and referred to among otherwise unrelated systems.
+## FAQ
 
-**Q: Is there any recommendation for the identifier?**
+**Q: Why is a PAC-ID represented as an HTTPS URL?**
 
-**A:** Basically the `identifier` is almost free to use with any format you'd like. Also, you can just reuse your existing identifiers. Other than that it's recommended to use a 128-bit UUID as identifier (encoded as base64url according to [RFC 4648](https://www.ietf.org/rfc/rfc4648.html)).
+**A:** The biggest advantage is that PAC-IDs can directly be entered as address into a browser (or scanned via QR code for example) on almost any device. To be able to easily recognize a PAC-ID as such, it was decided to use the subdomain `PAC.` instead.
```

Comment:
"instead" can probably removed. otherwise make more clear that we could have used PAC:// instead